packaging/nnstreamer.spec

```diff
@@ -273,6 +286,11 @@ popd
 %defattr(-,root,root,-)
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_tensorflow-lite.so
 
+%files -n nnstreamer-python2
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_python2.so
+%{_prefix}/lib/nnstreamer/filters/nnstreamer_python2.so
```

Comment:
Have you tested if ```%{_prefix}/lib/nnstreamer/filters/nnstreamer_python2.so``` is dynamically-linkable from applications or the filter? You need to test with another packages that uses nnstreamer (e.g., nnstreamer.git) to test it. You will not be able to test it ("integration-test") with simply gbs-building nnstreamer.